docs/developer-guide/working-with-plugins.md

```diff
@@ -140,6 +140,11 @@ ruleTester.run("custom-plugin-rule", rule, {
 });
 ```
 
+#### Customizing RuleTester
+
+RuleTester internally uses Mocha's `describe` and `it` methods when available to create tests for each valid and invalid case. If you use another test runner, you can override `RuleTester.describe` and `RuleTester.it` to make RuleTester compatible with it and have proper individual tests and feedback.
```

Comment:
Point 1 and 3 sound good.

Not sure I get point 2 though. Are you saying I should leave out the word `internally` as it may already be implied? I don't think it hurts to make this explicit.

What about @sindresorhus's comment above showing an example? I think having an example is nice, and my package is working well (in "production" in 2 plugins already), but I don't get the feeling that there a links to a lot of external packages that are not in the ESLint org (and thus out of your control?).